grug/types/src/app.rs

```diff
@@ -97,6 +97,13 @@ pub struct BlockInfo {
     pub hash: Hash256,
 }
 
+#[derive(Serialize, Deserialize, BorshSerialize, BorshDeserialize, Debug, Clone, PartialEq, Eq)]
+#[serde(deny_unknown_fields)]
+pub struct Block {
+    pub block_info: BlockInfo,
+    pub txs: Vec<Tx>,
+}
```

Comment:
If possible try to avoid to create this container struct. We already have a lot of types, create new types only if needed (in this case Block is only used to wrap `block_info` and `txs`, not a big deal)

Reply:
I only did as @larry0x suggested at https://discord.com/channels/1246466309080879207/1260999513859166342/1316133794734149712:

```
Larry Engineer: we should do
struct Block {
    pub info: BlockInfo,
    pub txs: Vec<Tx>,
}
```

Should I have added it somewhere else?

Reply:
Ok so